src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -1618,6 +1618,12 @@
             "title": "Discard pending console input on error",
             "type": "boolean",
             "default": true
+        },
+        "editor_scroll_multiplier": {
+            "description": "A integer value, to set the editor scroll multiplier.",
```

Comment:
Maybe a little parenthetical here to indicate higher means more sensitive or something?

Reply:
Yes, that would be good!